### PR TITLE
Dom api task

### PR DESCRIPTION
**Pull request by Zenzelya**
### DOM-API-TASK ptp group

[Demo](https://zenzelya.github.io/dom/dist/) |
[Code base](https://github.com/Zenzelya/frontend-2019-p2p/tree/dom-api-task/submissions/zenzelya/dom-api-task)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/zenzelya/dom-api-task/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">		
+  <title>Webpack</title>
```
Comment (aymkin) [inline submissions/zenzelya/dom-api-task/dist/index.html]: I wouldn't like to see webpack title

File: submissions/zenzelya/dom-api-task/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			<li data-list="top_rated" 	class="movies-list__item">Top Rated movies</li>	
+			<li data-list="popular" 	class="movies-list__item">Most Popular</li>
+			<li data-list="play_now" 	class="movies-list__item">In Cinema</li>
+		</ul>
+		<main class="card__wrapper js-hook--card-wrapper">
+		</main>
+	</section> 
+	
+<script type="text/javascript" src="assets/js/app.js"></script></body>
+</html>
```
Comment (aymkin) [inline submissions/zenzelya/dom-api-task/dist/index.html]: Looks like the empty line at the end of the file needed/

File: submissions/zenzelya/dom-api-task/dist/index.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">		
+  <title>Webpack</title>
+<link href="assets/css/app.css" rel="stylesheet"></head>
+<body>
+	<section class="page-wrapper">
+		<ul class="movies-list" id='js-hook--movies-list'>
+			<li data-list="upcoming" 	class="movies-list__item movies-list__item--selected">Upcoming movies</li>
```
Comment (aymkin) [inline submissions/zenzelya/dom-api-task/dist/index.html]: I would like to use tabs with 2 spaces

File: submissions/zenzelya/dom-api-task/dist/index.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```
Comment (aymkin) [inline submissions/zenzelya/dom-api-task/dist/index.html]: Please, upload the src files only, do not upload the build files

File: submissions/zenzelya/dom-api-task/src/js/index.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+// another js file (example)
```
Comment (aymkin) [inline submissions/zenzelya/dom-api-task/src/js/index.js]: Why do you upload two files `index.js` in the same directory? Please, get rid of this

File: submissions/zenzelya/dom-api-task/src/scss/modules/footer.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+// footer style
```
Comment (nazmariam) [inline submissions/zenzelya/dom-api-task/src/scss/modules/footer.scss]: in some smart books authors wrote that you should never write code in advance
this footer style is redundant

File: submissions/zenzelya/dom-api-task/src/scss/modules/header.scss [hunk 7]
```diff
@@ -0,0 +1 @@
+// header style
```
Comment (nazmariam) [inline submissions/zenzelya/dom-api-task/src/scss/modules/header.scss]: same here

**Pull request closed without merging**